### ISSUE TITLE
Support Workspaces

### ISSUE THREAD

**Issue by ddunkin** [mrward/monodevelop-nuget-extensions#5]
When multiple solutions are loaded in a workspace, the extension only works with the last loaded solution.

**Comment by mrward** [comment]
Yeah this is a known bug which also happens with some parts of the NuGet addin that is included in the box with MonoDevelop.
